integrations-and-supported-tools/pytorch-ignite/notebooks/Neptune_PyTorch_Ignite.ipynb

```diff
@@ -0,0 +1,704 @@
+    "    tag=\"validation\",\n",
+    "    metric_names=\"all\",\n",
+    "    global_step_transform=global_step_from_engine(\n",
+    "        trainer\n",
+    "    ),  # takes the epoch of the trainer instead of train_evaluator\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Erd4Xi3Gelun"
+   },
+   "source": [
+    "## (Neptune) Log optimizer parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "WlYJPi3jelun"
+   },
+   "outputs": [],
+   "source": [
+    "neptune_logger.attach_opt_params_handler(\n",
+    "    trainer,\n",
+    "    event_name=Events.ITERATION_COMPLETED(every=100),\n",
+    "    optimizer=optimizer,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "fMVUNWmhelun"
+   },
+   "source": [
+    "## (Neptune) Log model's normalized weights and gradients after each iteration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "K9p9HsQoeluo"
+   },
+   "outputs": [],
+   "source": [
+    "from ignite.contrib.handlers.neptune_logger import WeightsScalarHandler\n",
+    "\n",
+    "neptune_logger.attach(\n",
+    "    trainer,\n",
+    "    log_handler=WeightsScalarHandler(model, reduction=torch.norm),\n",
+    "    event_name=Events.ITERATION_COMPLETED(every=100),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "8BGG8Wi-elup"
+   },
+   "outputs": [],
+   "source": [
+    "from ignite.contrib.handlers.neptune_logger import GradsScalarHandler\n",
+    "\n",
+    "neptune_logger.attach(\n",
+    "    trainer,\n",
+    "    log_handler=GradsScalarHandler(model, reduction=torch.norm),\n",
+    "    event_name=Events.ITERATION_COMPLETED(every=100),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "zc-OyA5pelup"
+   },
+   "source": [
+    "## (Neptune) Save model checkpoints\n",
+    "__Note:__ `NeptuneSaver` currently does not work on Windows"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "r38cOqM5elup"
+   },
+   "outputs": [],
+   "source": [
+    "from ignite.handlers import Checkpoint\n",
+    "from ignite.contrib.handlers.neptune_logger import NeptuneSaver\n",
+    "\n",
+    "\n",
+    "def score_function(engine):\n",
+    "    return engine.state.metrics[\"accuracy\"]\n",
+    "\n",
+    "\n",
+    "to_save = {\"model\": model}\n",
+    "\n",
+    "handler = Checkpoint(\n",
+    "    to_save=to_save,\n",
+    "    save_handler=NeptuneSaver(neptune_logger),\n",
+    "    n_saved=2,\n",
+    "    filename_prefix=\"best\",\n",
+    "    score_function=score_function,\n",
+    "    score_name=\"validation_accuracy\",\n",
+    "    global_step_transform=global_step_from_engine(trainer),\n",
+    ")\n",
+    "\n",
+    "# validation_evaluator.add_event_handler(Events.COMPLETED, handler)"
```

Comment:
Probably add another comment saying uncomment to save model checkpoint. On quick reading it seems that this cell should save checkpoints.